docs/developer/first-transform.md

```diff
@@ -0,0 +1,148 @@
+As we mentioned elsewhere, [setting up a generator](/developer/setting-up-generator/) is a great way to establish a repository template from which all of your newer projects can derive. There's one more powerful function you can apply when generating a project, and that's a _code transform_.
+
+A code transform allows you to change any of the files in a seed repository at the time of a project's generation. You can use it to do anything from changing the copyright year in source code header licenses to manipulating the AST of the files entirely.
+
+In this example, we'll apply a code transform that does two things to a Node project: first, it'll update the README.md to note the project's name, and second, it'll also update the package.json file to use the project's name, too.
+
+## Prerequisites
+
+You should already have [a blank SDM](/developer/sdm/#creating-an-sdm-project), though you can also use an existing one if you'd like.
+
+We'll be using the [atomist-seeds/express-es6-rest-api](https://github.com/atomist-seeds/express-es6-rest-api) repository as our seed. Please don't use this seed for your own Express projects, as this is just a demo app (and likely full of outdated packages!).
```

Comment:
is that a problem? can we keep this seed up?

Reply:
I mean, we could drop that parenthetical, but I think it's unrealistic to expect that the seed will be maintained by Atomist, for example, a node package has a CVE. The seed is just there for an example and I don't want to give the impression that it should be relied upon for a production app.